hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -5,7 +5,7 @@ title: "SRAM_CTRL DV document"
 ## Goals
 * **DV**
   * Verify all SRAM_CTRL IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules, except prim_prince and prim_lfsr, which are verified in separated testbenches.
```

Comment:
Perhaps we should add a separate heading for this:
```
## Pre-verified sub-modules

The following sub-modules are pre-verified:
* [prim_prince](link-to-tb/reports)
* [prim_lfsr](...)

Only toggle coverage on the IOs of these sub-modules is enabled for coverage collection. 

```

Reply:
Good idea. Updated. Thanks